app/views/posts/show.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>List of post for specific user</h1>
-
+<div class='mt-3 d-flex align-items-center flex-column'>
+  <div class="card mb-3 border-primary" style="width: 670px;">
+    <div class="card-header bg-primary text-white">
+      <div class='row'>
+        <div class='col-md-8'><%= @posts.title %> by <%= @user.name %></div>
+        <div class='col-md-4 d-flex gap-2'>
+          <p class="card-text"><small class="text-white">Comments: <%= @posts.comment_counter %></small></p>
+          | 
+          <p class="card-text"><small class="text-white">Likes: <%= @posts.likes_counter %></small></p>
+        </div>
+      </div>
+    </div>
+    <div class="card-body">
+      <p class="card-text"><%= @posts.text%></p>
+    </div>
+  </div>
+<h6>Recent Comments</h6>
```

Comment:
- [ ] We are not really rendering the recent comments of a post on the show page, we are doing that on the post index page instead. To avoid confusion, kindly remove the heading.